cypress/e2e/yves/ssp-service/ssp-service.cy.ts

```diff
@@ -0,0 +1,178 @@
+import {container} from '@utils';
+import {CustomerLoginScenario, CustomerLogoutScenario, CheckoutScenario} from '@scenarios/yves';
+import {SspServiceListPage, CatalogPage, ProductPage} from '@pages/yves';
+
+(['suite'].includes(Cypress.env('repositoryId')) ? describe : describe.skip)(
+    'SSP Service Management',
+    {tags: ['@yves', '@ssp-service', '@ssp', '@SspServiceManagement']},
+    (): void => {
+        const customerLoginScenario = container.get(CustomerLoginScenario);
+        const customerLogoutScenario = container.get(CustomerLogoutScenario);
+        const sspServiceListPage = container.get(SspServiceListPage);
+        const checkoutScenario = container.get(CheckoutScenario);
+
+        let staticFixtures: Record<string, any>;
+        let dynamicFixtures: Record<string, any>;
+        let isSetupDone = false;
+
+        before((): void => {
+            ({staticFixtures, dynamicFixtures} = Cypress.env());
+        });
+
+        // Setup method to be called in each test
+        const prepareServices = () => {
```

Comment:
not a must-have, just a nice-to-have to follow the same style.

- more this method to the end of the file 
- use 'function' instead of creating methods via constants. 

See an example of how it could look in another comment